src/components/TextInput/TextInput.stories.mdx

```diff
@@ -0,0 +1,96 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+import { action } from '@storybook/addon-actions';
+import TextInput from './TextInput';
+
+<Meta title="TextInput" component={TextInput} />
+
+# TextInput
+
+A basic input component for either single or multiline input. Use component props to add a label above
```

Comment:
what makes it basic?

Reply:
Hmm I was just thinking of it as uncontrolled and not particularly laden with bells or whistles; fair point